README.md

```diff
@@ -1,4 +1,28 @@
 # Kotlin Wallet SDK
 
 Kotlin Wallet SDK is a library that allows developers to build wallet applications on the Stellar network faster. It
-utilizes [Java Stellar SDK](https://github.com/stellar/java-stellar-sdk) to communicate with a Stellar Horizon network.
\ No newline at end of file
+utilizes [Java Stellar SDK](https://github.com/stellar/java-stellar-sdk) to communicate with a Stellar Horizon network.
+
+## Wallet
+
+`class Wallet(horizonUrl: String, horizonPassphrase: String)`
+
+Example
+
+```kotlin
+// Using Horizon testnet configuration
+val wallet = Wallet("https://horizon-testnet.stellar.org", "Test SDF Network ; September 2015")
```

Comment:
We could use the Java SDK's hard-coded values
```suggestion
import org.stellar.sdk.Network;

val wallet = Wallet("https://horizon-testnet.stellar.org", Network.TESTNET.toString())
```

Reply:
Is our preferred way to use SDK defaults? I went with a simple string to avoid importing SDK just for that, but we can change it.

Reply:
I don't feel strongly about it, but those constants are there to help. I've found myself using them more often lately because I don't like having to search around to find the right string.